sycl/test-e2e/Basic/image/image_array.cpp

```diff
@@ -3,7 +3,7 @@
 // RUN: %{build} -o %t.out
 // RUN: %{run} %t.out
 
-// UNSUPPORTED: gpu
+// UNSUPPORTED: accelerator
```

Comment:
I think we should add `// REQUIRES: aspect-ext_intel_legacy_image` at line 1 instead. `gpu` was /is unsupported because its run line was `RUNx`.

Reply:
oh, I see, thanks!